config.py

```diff
@@ -73,6 +91,9 @@ class Config(object):
     UAA_CLIENT_ID = os.getenv('UAA_CLIENT_ID')
     UAA_CLIENT_SECRET = os.getenv('UAA_CLIENT_SECRET')
 
+    EMAIL_TOKEN_SALT = os.getenv('EMAIL_TOKEN_SALT', 'aardvark')
+    EMAIL_TOKEN_EXPIRY = int(os.getenv('EMAIL_TOKEN_EXPIRY', '306000'))
```

Comment:
Do we know if the value here is in seconds or miliseconds etc? And can we please put a comment above as to which one it is please? I also don't think that this equates to the 24h shown in the email template.

Reply:
It's in seconds and yeah, completely wrong - missed that A/C :) changing